### DISCUSSION TITLE
Pokédex terminada

### DISCUSSION THREAD

**Discussion by JossPerezT** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#535] [Show and tell]
Hola explores. 
Aquí les dejo mi el link de mi pokédex muy básico. Siento que le falta mucho pero le iré metiendo más funciones en cuanto tenga más tiempo y conocimientos. No es responsive pero si ponen el móvil en horizontal se ve mejor. 😁🚀

[]()https://jossperezt.github.io/JS-Pokedex/

**Comment by khisus19** [comment]
Esta genial. Te felicito!

**Reply by JossPerezT** [reply]
Gracias!! 😁
